### DISCUSSION TITLE
Shelly Mini 1PM G3.  Does HAA support it?

### DISCUSSION THREAD

**Discussion by juanfal** [RavenSystem/esp-homekit-devices#2266] [Q&A]
I have just bought a pair of them ([20€](https://amzn.eu/d/dGsc1Ka)) and wanted to integrate it at its most.  I understand it won't appear as power meter on HomeKit, but does **HAA Home Manager** show, accumulate its power consume measure.  And, where is the HAA Firmware for it?
Thanks!

**Comment by RavenSystem** [comment]
HAA has full support for Shelly Mini 1PM Gen3. Check this out: https://github.com/RavenSystem/mgos32toHAA
